cmd/root.go

```diff
@@ -62,12 +61,19 @@ func ExecuteContext(ctx context.Context) error {
 	return rootCmd.ExecuteContext(ctx)
 }
 
-func init() {
+func getKubeCfg() string {
```

Comment:
I know this codebase is not held to the same internal style guide, but worth mentioning: go/go-style/decisions#getters